src/itemButtonList.js

```diff
@@ -84,11 +95,41 @@ function itemButtonListFactory(config = {}) {
             if ($target.length) {
                 $target.addClass(cssClasses.active);
                 // finally make sure the item is visible
-                autoscroll($target, component.getElement());
+                autoscroll($target, getScrollContainer());
+
+                const $ariaTarget = component.getElement().find(cssSelectors.navigableById(itemId));
+                if ($ariaTarget.length) {
+                    $ariaTarget.attr('aria-current', 'location');
+                }
```

Comment:
I agree, this check here is useless. But I think I'll leave `if ($target.length) {` check above, because it has logical meaning - $target may really not exist, no one checks if `itemId` is valid or not.